src/eradius_server.erl

```diff
@@ -198,22 +198,26 @@ do_radius(ServerPid, ReqKey, Handler = {HandlerMod, _}, NasProp, {udp, Socket, F
             RequestHandletime = TS2 - TS1,
             exometer:update_or_create([eradius, server, ServerName, request_handle_time], RequestHandletime, histogram, [{truncate, false}]),
             gen_udp:send(Socket, FromIP, FromPort, EncReply),
-            ServerPid ! {replied, ReqKey, self()},
             eradius_metrics:update_nas_prop_metric(replies, NasProp, 1),
             eradius_counter:inc_counter(replies, NasProp),
-            {ok, ResendTimeout} = application:get_env(eradius, resend_timeout),
-            wait_resend_init(ServerPid, ReqKey, FromIP, FromPort, EncReply, ResendTimeout, ?RESEND_RETRIES);
+            case application:get_env(eradius, resend_timeout, 2000) of
```

Comment:
I would prefer to put this into the state. It is just an ets lookup, but kind of redundant.